xarray/core/dataset.py

```diff
@@ -8424,6 +8540,40 @@ def argmin(self: T_Dataset, dim: Hashable | None = None, **kwargs) -> T_Dataset:
         -------
         result : Dataset
 
+        Example
+        -------
+
+        # Defined the dataset
+        >>> dataset = xr.Dataset(
+        ...     {
+        ...         "math_scores": (
+        ...             ["student", "test"],
+        ...             [[90, 85, 79], [78, 80, 85], [95, 92, 98]],
+        ...         ),
+        ...         "english_scores": (
+        ...             ["student", "test"],
+        ...             [[88, 80, 92], [75, 95, 79], [93, 96, 78]],
+        ...         ),
+        ...     },
+        ...     coords={
+        ...         "student": ["Alice", "Bob", "Charlie"],
+        ...         "test": ["Test 1", "Test 2", "Test 3"],
+        ...     },
+        ... )
+
+        # Indices of the minimum values along the 'student' dimension are calculated
+        >>> argmin_indices = dataset.argmin(dim="student")
+
+        # Print the indices of the minimum values
+        >>> argmin_indices
+        <xarray.Dataset>
+        Dimensions:         (test: 3)
+        Coordinates:
+        * test            (test) <U6 'Test 1' 'Test 2' 'Test 3'
+        Data variables:
+            math_scores     (test) int64 1 1 0
+            english_scores  (test) int64 1 0 2
+
```

Comment:
The purpose of argmin might be more obvious if you then used this result to index the original array, thus returning the names of the students.